capella2polarion/converters/element_converter.py

```diff
@@ -57,6 +67,16 @@ def _format(texts: list[str]) -> dict[str, str]:
     return requirement_types
 
 
+def _resolve_capella_attribute(
+    element: m.ModelElement | m.Diagram, attribute: str
+) -> polarion_api.TextContent:
+    value = getattr(element, attribute)
+    if isinstance(value, enum.Enum):
+        return polarion_api.TextContent(type="string", value=value.name)
+
+    raise ValueError(f"Unsupported attribute type: {value!r}")
```

Comment:
I think we should be more generic here. We could add an HTML-Flag to the config of an attribute. If this flag is set, we export the value of the attribute as `HtmlContent`. If not, we just cast the value to string and use the string